drivers/display/Kconfig.st7735r

```diff
@@ -0,0 +1,26 @@
+# ST7735R display driver configuration options
```

Comment:
Remove first two lines.

Reply:
The majority of Kconfigs with in drivers/display have those two first lines.